wolfcrypt/src/port/st/stsafe.c

```diff
@@ -322,7 +322,11 @@ int SSL_STSAFE_SharedSecretCb(WOLFSSL* ssl, ecc_key* otherKey,
     }
 
     /* Compute shared secret */
-    err = stsafe_interface_shared_secret(curve_id, &otherKeyX[0], &otherKeyY[0],
+    err = stsafe_interface_shared_secret(
+#ifdef WOLFSSL_STSAFE_SHARED_SECRET_TAKES_SLOT
```

Comment:
Lets shorten this macro name, does it only apply to shared secret? Maybe could be WOLFSSL_STSAFE_TAKES_SLOT

Reply:
Currently it's shared secret only but it may be expanded in the future.  Shortened.